src/python/grpcio/grpc/_channel.py

```diff
@@ -926,6 +926,11 @@ def __init__(self, target, options, credentials):
         self._call_state = _ChannelCallState(self._channel)
         self._connectivity_state = _ChannelConnectivityState(self._channel)
 
+        # TODO(https://github.com/grpc/grpc/issues/9884)
```

Comment:
Typography nitpick: missing colon after closing parenthesis and start the prose of the comment on the same line rather than line-wrapping after the `TODO(...):`.

[Huh. I guess the colon isn't required.](https://google.github.io/styleguide/pyguide.html?showone=TODO_Comments#TODO_Comments) Aww phooey; I've always liked it.